README.md

```diff
@@ -210,7 +210,20 @@ The code below will enable push notifications to show up when you app is running
 
 If your user taps on the notification this will be tracked back to Klaviyo as an "Opened Push" event assuming you have implemented the tracking changes discussed above.
 
+## SDK Data Transfer
 
+As version 1.7.0 the SDK will cache incoming data and flush it back to the Klaviyo API on an interval. As of this writing the interval is based on the network link currently being used by the app. The table below shows the flush interval used for each type of connection:
+
+| Network     | Interval    |
+| :---        | :--- |
+| WWAN/Wifi   | 10 seconds  |
+| Cellular    | 30 seconds  |
+
+Connection determination is based on notifications from our reachability service. When there is no network available the SDK will cache data until the network becomes available again. All data sent by the SDK should be available shortly after it is flushed by the SDK. 
+
+### Retries
+
+The SDK will retry API requests that fail under conditions. For example if a network timeout occurs the request will be retried on the next flush inteval. In addition if the SDK received a 429 from the Klaviyo API it will use expontential backoff with jitter to retry the next request.
```

Comment:
Should it be "fail under certain conditions"?

Also even though most people will know what a 429 error is, I might specify that it's a rate limiting error... In addition, if the SDK received a 429 rate limiting error from the Klaviyo API, it will use exponential backoff with jitter to retry the next request.

Reply:
```suggestion
The SDK will retry API requests that fail under certain conditions. For example if a network timeout occurs the request will be retried on the next flush interval. In addition if the SDK receives a rate limiting error (429) from the Klaviyo API it will use exponential backoff with jitter to retry the next request.
```